benchmarks/memcached/run.sh

```diff
@@ -122,16 +122,18 @@ done
 rm -rf results
 mkdir -p results
 
-docker run --name memcached-docker -d --privileged $MAXMEM $MAXSWAP $CPUSETS_CPU $CPUSETS_MEM -p 11211:11211 -u 11211 -it $CONTAINER numactl $MEMPOLICY memcached -m 262144
+
+docker run --name memcached-docker -d --privileged $MEMPOLICY $MAXMEM $MAXSWAP $CPUSETS_CPU $CPUSETS_MEM --network=host -u 11211 -it $CONTAINER -m 262144
```

Comment:
Ideally, we would like to verify this command succeeds before proceeding and handle any errors appropriately.